Wabbajack/Views/MainWindow.xaml

```diff
@@ -81,4 +81,11 @@
             </Button>
         </mahapps:WindowCommands>
     </mahapps:MetroWindow.RightWindowCommands>
+    <Window.TaskbarItemInfo>
+        <TaskbarItemInfo x:Name="taskBarItemInfo1" 
+                     ThumbnailClipMargin="80,0,80,140"
+                     Description="Progress"
```

Comment:
I think you need to set the "ProgressState" variable.  Default will not have any progress mechanics active